.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: "💬 Chat"
+    url: https://matrix.to/#/#jenkins-release:libera.chat
+    about: Please ask and answer questions here.
   - name: "❓ IRC"
```

Comment:
I don't think we need IRC and chat links? can't we just have the matrix one.

Reply:
Daniel wants to keep the IRC one, see the comment above. But in my opinion, matrix-only is fine.

Reply:
can you link to the comment? I'm not seeing it

Reply:
https://github.com/jenkins-infra/release/pull/289#discussion_r974677491

Reply:
I think that was referring to the text `Please ask and answer questions here` i.e. text not being useful. re-including #jenkins-release should fix that.